apps/sms/views/conversation/js/conversation.js

```diff
@@ -549,11 +551,13 @@ var ConversationView = {
     // TODO should we implement hooks to Navigation so that Threads could
     // get an event whenever the panel changes?
     Threads.currentId = args.id;
+    // TODO: Will need to replace with Conversation service method.
+    this.activeThread = Threads.active;
```

Comment:
so I see that `Threads.active` is used only here, and we only assign to `Threads.currentId` and read only in `Threads.active` accessor...

So what do you think of removing both `Threads.active` and `Threads.currentId`, and replace with `Threads.get(args.id)` here ?

If you think this should be in a separate patch, it could be, but I think it's just as easy to do it now here, and this is quite the same purpose. What do you think ?